### ISSUE TITLE
Spyder 2.3.0beta3 (Win-64bit) crashes, when ipython is linked (miniconda)

### ISSUE THREAD

**Issue by spyder-issues-migrator** [spyder-ide/spyder#1729]
_From [wstra...@web.de](https://code.google.com/u/104871799641599986641/) on 2014-02-17T08:24:41Z_

Please note that reporting bugs from Spyder's help menu ("?" > "Report
issue...") will fill version details for Spyder, Python and Qt in this description for you. 

What steps will reproduce the problem?

 1. Installation of current Miniconda version on Windows 7 64-bit operating system (python 3.3.4 64 bit) from: http://conda.pydata.org/miniconda.html Miniconda3-3.0.4-Windows-x86_64

2. Installatiion of the following packages through conda: numpy, scipy, pandas, matplotlib, ipython (for version see below)

3. Installation of spyder 2.3.0beta3 from: http://www.lfd.uci.edu/~gohlke/pythonlibs/ spyder-2.3.0beta3.win-amd64-py3.3 What is the expected output? What do you see instead? Without having ipython 1.1.0 installed, spyder works fine.
However with ipython 1.1.0 linked, spyder crashes during launching.
(When unlinking ipython, spyder works fine again. What version of the product are you using? On what operating system? Windows 7, 64-bit operating system
spyder-2.3.0beta3
Miniconda3-3.0.4-Windows-x86_64
python 3.3.4 64 bit
ipython 1.1.0 

Please provide any additional information below

. List of installed python packages:

C:\Users\myadmin>conda list
\# packages in environment at C:\Miniconda3:
\#
conda                     3.0.4                    py33_0
dateutil                  2.1                      py33_2
ipython                   1.1.0                    py33_1
matplotlib                1.3.1                np18py33_1
numpy                     1.8.0                    py33_0
pandas                    0.13.1               np18py33_0
pycosat                   0.6.0                    py33_1
pyparsing                 2.0.1                    py33_0
pyreadline                2.0                      py33_0
pyside                    1.2.1                    py33_0
python                    3.3.4                         0
pytz                      2013b                    py33_0
pyyaml                    3.10                     py33_1
scipy                     0.13.3               np18py33_0
setuptools                2.2                      py33_0
six                       1.5.2                    py33_0

Please find enclose a screenshot of the error message and the installation protocol for miniconda including packages.

**Attachment:** [Miniconda insatllation for Python 3.3.4.txt Screenshot - spyder error message.pdf](http://code.google.com/p/spyderlib/issues/detail?id=1729)

_Original issue: http://code.google.com/p/spyderlib/issues/detail?id=1729_

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2014-02-17T12:26:03Z_

Could you try to start spyder from the terminal (i.e. cmd.exe) and look for any errors printed there?

It's not possible for me to help you right now with the info you gave us because I really don't know what can be happening.

**Labels:** MS-v2.3.1 OpSys-Windows

**Comment by spyder-issues-migrator** [comment]
_From [wstra...@web.de](https://code.google.com/u/104871799641599986641/) on 2014-02-17T15:20:56Z_

Hi,

Thank you very much for your quick response.

I was playing around in the meantime.
First,it seems in addition to the installed modules by miniconda, and installation of numpy, scipy, matplotlib, pandas and ipython
2 further modules are required:
- pygments
- pyzmq 

This I could read out from the spyder_crash.txt file. The cmd-console crashed as well, when starting spyder from it.
Then I was able to start spyder (but did not test its functionality).

Second, after installation of anaconda through miniconda ("conda insatall anaconda"), it worked fine as well.

Thus, very likely, that the installation of pygments and pyzmq might be sufficient to prevent the crash of spyder

Again, I have attached some files, that document the installation.

Best,

W.

**Attachment:** [spyder_crash_1.txt spyder_crash_2.txt installation pygments + pyzmq modules.txt Miniconda- Anaconda installation for Python 3.3.4__2.txt](http://code.google.com/p/spyderlib/issues/detail?id=1729#hc2)
